tf_agents/cogment_verse_tf_agents/reinforce/reinforce.py

```diff
@@ -30,48 +31,42 @@ class ReinforceAgent:
     # pylint: disable=too-many-instance-attributes
     def __init__(
         self,
-        id,
-        obs_dim,
-        act_dim,
-        epsilon=0.01,
-        gamma=0.95,
-        lr=3e-4,
-        max_replay_buffer_size=50000,
-        seed=42,
-        epsilon_schedule=None,
-        lr_schedule=None,
+        **params
     ):
-        self.id = id
-        self._params = {}
-        self._params["obs_dim"] = obs_dim
-        self._params["act_dim"] = act_dim
-        self._params["seed"] = seed
-        self._params["max_replay_buffer_size"] = max_replay_buffer_size
-        self._params["gamma"] = gamma
 
-        self._epsilon_schedule = epsilon_schedule
-        if epsilon_schedule is None:
-            self._epsilon_schedule = ConstantSchedule(epsilon)
+        self._params = {}
+        for k, v in params.items():
+            self._params[k] = v
```

Comment:
```suggestion
        self._params = params
```
isn't that the same thing?